scripts/fetch-iana.js

```diff
@@ -94,7 +94,7 @@ async function addTemplateData (data, options) {
     return
   }
 
-  let res = await got('https://www.iana.org/assignments/media-types/' + data.template)
+  let res = await got(`https://www.iana.org/assignments/media-types/${data.template}`, { throwHttpErrors: false })
```

Comment:
Actually we don't support new ES6+ syntax based on the CI: https://github.com/jshttp/mime-db/blob/master/.github/workflows/ci.yml#L13

Reply:
This script has es6 via async/await and let already, so the template string isn't a breaking introduction.

Interesting wrinkle here though, in that these scripts are part of the build tooling and we don't document the node version requirement for development/building. v11 of got has engines set to `10.19.0` https://github.com/sindresorhus/got/blob/1f6ac4597b797e6fe760a7dc11a3db8bf298aa94/package.json#L10C1-L11C1

I set up the automation PR that runs the build step on cron to use LTS node, so es6 is fine here.

https://github.com/jshttp/mime-db/blob/49f8df0e170c7b40785e8aa4f464793b7fcfa41b/.github/workflows/generate.yml#L23C1-L24C1